addons/website_links/static/src/js/website_links.js

```diff
@@ -31,7 +31,7 @@ var SelectBox = publicWidget.Widget.extend({
         defs.push(this._rpc({
             model: this.obj,
             method: 'search_read',
-            params: {
+            kwargs: {
```

Comment:
:eyes:

Reply:
This is another nasty-bug. again reveled by the warning-level in route_wrapper.